### DISCUSSION TITLE
约定式路由和配置式路由可以一起用吗， 如何操作

### DISCUSSION THREAD

**Discussion by ludonghai715** [umijs/umi#12423] [General]
约定式路由和配置式路由可以一起用吗， 如何操作

**Comment by fz6m** [comment]
不可以，只能二选一，但是无论哪种路由，都可以在运行时 patch 路由表，详见：[patchClientRoutes](https://umijs.org/docs/api/runtime-config#patchclientroutes) 。

demo：https://github.com/umijs/umi/discussions/9717#discussioncomment-4078053
